app/src/main/res/values/strings.xml

```diff
@@ -3,19 +3,19 @@
     <string name="main_activity_text_view_title">EstacionAR</string>
     <string name="main_activity_button_set_parking">Establecer estacionamientos</string>
     <string name="fragment_configure_parking_lots_title">Establecer estacionamientos</string>
-    <string name="fragment_btn_ok">OK</string>
-    <string name="fragment_btn_cancel">Cancelar</string>
+    <string name="fragment_button_ok">OK</string>
+    <string name="fragment_button_cancel">Cancelar</string>
```

Comment:
rename this id's pls

Reply:
these*

Reply:
"fragment_" removed since ok and cancel button are used trough various fragments and activities

Reply:
renamed